spec/schemas/_common.query_dsl.yaml

```diff
@@ -625,8 +625,16 @@ components:
               $ref: '_common.yaml#/components/schemas/GeoDistanceType'
             validation_method:
               $ref: '#/components/schemas/GeoValidationMethod'
+            ignore_unmapped:
```

Comment:
Add to CHANGELOG. This is a new field added.

Do you know which version this new option was added in? Might need `x-added-...`.

Reply:
I think this field has been inherited since 1.0. Is there an easy way to make sure (other than running the query against 1.0 :)?

Reply:
I usually look at code. We now have 1.3.x tests here, so this will fail if it didn't exist.